{{cookiecutter.python_name}}/{{cookiecutter.python_name}}/_version.py

```diff
@@ -1,2 +1,9 @@
-version_info = (0, 1, 0)
-__version__ = ".".join(map(str, version_info))
+import json
+import os.path as osp
+
+HERE = osp.abspath(osp.dirname(__file__))
+
+with open(osp.join(HERE, 'package.json')) as fid:
+    data = json.load(fid)
+
+__version__ = pdata['version']
```

Comment:
This seems to be a typo:

```suggestion
__version__ = data['version']
```